frontend/src/components/InnerNavbar/index.tsx

```diff
@@ -0,0 +1,18 @@
+/***** CONSTS *****/
+import React from 'react';
```

Comment:
Love that you picked up on the code comments that I've been adding in files. If possible, move the react import under it's own heading called `/***** BASE IMPORTS *****/`

Basically, anything that comes from an external package such as react, tanstack router, zod, classnames, etc. should go under base imports

Reply:
Looks like this one was missed.